musicalbum.rb

```diff
@@ -0,0 +1,18 @@
+require './item'
+
+class Musicalbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(on_spotify)
+    super(publish_date)
```

Comment:
- [ ] I think the initialize method should have publish_date parameter. e.g. `def initialize(publish_date, on_spotify)`